api/swagger.yml

```diff
@@ -508,7 +508,11 @@ components:
           type: string
       required:
         - id
-
+    SetupState:
+      type: object
+      properties:
+        initialized:
+          type: boolean
```

Comment:
I don't like booleans for something like this, because they have only 2 values...  E.g. suppose we had a setup _wizard_, and you could be half set up.

Could we instead have an enum (i.e. a string with known names)?